tests/mm0_mmu/fail/bad-int1.mm0

```diff
@@ -0,0 +1,2 @@
+--                      V--  overflowing hazard
+prefix p : $+$ prec 4294967296;
```

Comment:
this is a `run` test, verifiers may or may not reject it

Reply:
what does it mean for the test, should we remove it ? 

Developpers should fail on such a test (rejecting bad input), right ?